src/test/versionbits_tests.cpp

```diff
@@ -290,6 +312,93 @@ BOOST_AUTO_TEST_CASE(versionbits_sanity)
     }
 }
 
+/** Check that ComputeBlockVersion will set the appropriate bit correctly for BIP8 (height-based) activation */
+static void check_computeblockversion_bip8(const Consensus::Params& params, Consensus::DeploymentPos dep)
```

Comment:
Please thoroughly review this to ensure the miner is configured to signal as expected for each state.